README.md

```diff
@@ -1,5 +1,7 @@
 # unleash
 
+__Warning: We are in the process of splitting up unleash into multiple packages in this repository, if you want to test the previous package see [previous tag](https://github.com/finn-no/unleash/tree/v1.0.0-alpha.1) __ 
```

Comment:
ok, but its partly packages - so this is from the state before splitting.

Btw, Im rethinking if its best to release 1.0.0....